contracts/lib/IdentityBase.sol

```diff
@@ -9,6 +9,18 @@ import "../lib/OnChainIdentity.sol";
 //  * @dev Contract managing onchain identity
 //  */
 contract IdentityBase {
+    struct IssuerStates {
+        uint256 state;
+        uint256 rootOfRoots;
+        uint256 claimsTreeRoot;
+        uint256 revocationTreeRoot;
+    }
+
+    struct CredentialStatus {
+        IssuerStates issuer;
+	    SmtLib.Proof mtp;
+    }
```

Comment:
I think is github issue. Since on editor all OK
<img width="242" alt="зображення" src="https://github.com/iden3/contracts/assets/20016700/970e0a33-5812-489a-998a-a611b4a407d0">